x-pack/legacy/plugins/maps/public/layers/sources/client_file_source/geojson_file_source.js

```diff
@@ -107,8 +107,14 @@ export class GeojsonFileSource extends AbstractVectorSource {
   }
 
   async getGeoJsonWithMeta() {
+    const { features, type } = this._descriptor.featureCollection;
+    const featuresWithoutProps = features.map(({ type, geometry }) =>
+      ({ type, geometry, properties: {} }));
     return {
-      data: this._descriptor.featureCollection,
+      data: {
+        type,
```

Comment:
this is a type-constant , and would read better as `type: 'FeatureCollection'` imo.